### PR TITLE
Add support Svadu extension

### PR DESCRIPTION
**Pull request by ved-rivos**
Add hade/stce/pbmte bits to menvcfg
Add menvcfgh with hade/stce/pbmte bits.

### CODE REVIEW

**Comment by neelgala** [comment]
#137 needs to be part of this PR

File: riscv_config/schemas/schema_isa.yaml [hunk 1]
```diff
@@ -1444,7 +1444,7 @@ hart_schema:
               schema:
                 description: { type: string, default: control the endianness of memory accesses other than instruction fetches for machine mode}
                 shadow: {type: string, default: , nullable: True}
-                shadow_type: {type: string, default: rw, nullable: True, allowed: ['rw','ro']}
+                3hadow_type: {type: string, default: rw, nullable: True, allowed: ['rw','ro']}
```
Comment (neelgala) [inline riscv_config/schemas/schema_isa.yaml:1447]: typo - the `s` is replaced with a `3`

Comment (ved-rivos) [reply riscv_config/schemas/schema_isa.yaml]: fixed the typo.

**Comment by ved-rivos** [comment]
I closed the other PR #137 and moved the contents to this PR. Also as you might notice the AR changed the extension name yesterday from Svadu to Svhad.

**Comment by ved-rivos** [comment]
AR changed it back to Svadu.

**Review by neelgala — APPROVED** [review]

**Pull request merged**